### ISSUE TITLE
i have a question

### ISSUE THREAD

**Issue by YinXiaojieCSDN** [fabienbaradel/object_level_visual_reasoning#4]
Hello,I run your project,but i can only get maskRCNN__png. I can not get other that you show your result in your article.I want to know how i need to do.

**Comment by fabienbaradel** [comment]
Hi @YinXiaojieCSDN,
What do you mean by "maskRCNN__png"? Sorry but I do not understand what would you like to do?
Do you want to do transfer learning on other datasets?
Best,

**Comment by YinXiaojieCSDN** [comment]
I want to do transfer learning on other datasets in future. Now I want to run your project to know what effect I can get through your project.I run your project followed your prompts.  I do preprocessing  ,training ,testing.I only get masks,and the masks are very   blurry.Why I can not get the semantics in a video or between pictures.

**Comment by fabienbaradel** [comment]
Okay I have now a big picture of what you want to do, thanks.
Yes the masks are not perfect because they are estimated and do not correspond to ground-truth masks. You can increase the confidence threshold from 0.5 to 0.8 if you want to handle only less blurry masks. By default, I have included objects that have been detected with at least a confidence of 0.5.
What do you mean by "getting the semantics in a video or between pictures"? You want to get the predicted classes for each video?
In this case you go here https://github.com/fabienbaradel/object_level_visual_reasoning/blob/master/inference/train_val.py#L184 , the variables ```output```corresponds to the predictions for the whole video.

About transfer learning, you should first run MaskRCNN on the new dataset using for example the official version. You can look at this repo https://github.com/facebookresearch/maskrcnn-benchmark
Once you have the masks data you have to create a new dataloader and then you can use the current code to train on your dataset.

I hope that it helps you.

**Comment by YinXiaojieCSDN** [comment]
Thank you , so ,how i can get the output?i print it ,but i get nothing
